middleware/multer.js

```diff
@@ -0,0 +1,18 @@
+import multer from "multer";
+import path from "path";
+
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, "uploads/");
+  },
```

Comment:
uploads 디렉토리가 없으면 에러가 발생할 것 같아요.
node의 filesystem을 사용해서, 디렉토리가 없다면 자동으로 생성하는 로직을 추가해보시는 것이 좋겠습니다.

```suggestion
import multer from "multer";
import path from "path";
import fs from "fs";

const storage = multer.diskStorage({
  destination: function (req, file, cb) {
    const uploadDir = 'uploads/';
    if (!fs.existsSync(uploadDir)) {
      fs.mkdirSync(uploadDir, { recursive: true });
    }
    cb(null, uploadDir);
  }
```